ocs_ci/utility/utils.py

```diff
@@ -1039,6 +1039,29 @@ def get_csi_versions():
     return csi_versions
 
 
+def get_ocp_version(seperator=None):
+    """
+    Get current ocp version
+
+    Args:
+        seperator (str): String that would seperate major and
+        minor version nubers
+
+    Returns:
+        string : If seperator is 'None', version string will be returned as is
+        eg: '4.2', '4.3'.
+        If seperator is provided then '.' in the version string would be
+        replaced by seperator and resulting string will be returned.
+        eg: If seperator is '_' then string returned would be '4_2'
+
+    """
+    char = seperator if seperator else '.'
+    log.info(config.DEPLOYMENT['installer_version'])
```

Comment:
I do not think we need to log anything here and at least not with info level. If you insist from some reason on logging the version you should use debug level but I would remove this line.

Reply:
sorry , I had it for debugging forgot to cleanup